jupyterlab_nodeeditor/yggdrasil_support.py

```diff
@@ -1,46 +1,49 @@
-import uuid
-
+from yggdrasil.examples import yamls as ex_yamls
 import yaml
+import jupyterlab_nodeeditor as jlne
+import pytest
+
+# Improved version of making a JLNE-compliant dictionary from a Yggdrasil Model YAML
+# Still semi-hard coded for the Photosynthesis model
+# TO-DO : Make more robust, Test on custom models 1-3
+def dict_conversion(model_dict):
+    # Setup initial dictionary to be filled
+    new_dict, new_dict["inputs"], new_dict["outputs"], new_dict["title"] = {}, [], [], model_dict["name"]
+    
+    # Fill in the Inputs
+    for i, inp in enumerate(model_dict["inputs"]):
+        new_dict["inputs"].append({'title': inp["name"], 'key': "temp_in" + str(i), 'socket_type': inp["default_file"]["filetype"]})
```

Comment:
use format strings or f-strings

```suggestion
        new_dict["inputs"].append({'title': inp["name"], 'key': f"temp_in{i}", 'socket_type': inp["default_file"]["filetype"]})
```